worker/legacy/manager.cpp

```diff
@@ -15,6 +16,7 @@ using ava_manager::ManagerServiceServerBase;
 
 bool cfgWorkerPoolDisabled = true;
 uint32_t cfgWorkerPoolSize = 3;
+extern char** environ;
```

Comment:
This is actually not the right way to go. `environ` exposes too much information (of the scheduling node) to the worker (the execution node).

Reply:
Sure. I only want to expose CUDA environment variables to the worker.

Since the worker only executes CUDA APIs what is the harm in passing all environment variables?

Reply:
The worker is supposed to support more than CUDA APIs. But basically it's about isolation.